aws-android-sdk-appsync/src/main/java/com/amazonaws/mobileconnectors/appsync/AWSAppSyncClient.java

```diff
@@ -698,4 +701,22 @@ public boolean isCanceled() {
         return this.sync(baseQuery, baseQueryCallback, subscription, subscriptionCallback, null, null, 0 );
     }
 
+    /**
+     * Used to check if the mutation queue is empty.
+     * @return true if queue is empty, false otherwise.
+     */
+    public boolean mutationQueueEmpty() {
```

Comment:
should we use `isMutationQueueEmpty`? 

Another thought is to have a `MutationQueue` object which has these new operations. Something like:

```java
public class MutationQueue {
     public boolean isEmpty() {...}
     public void clear() {...}
}
```

Customer experience would be like:

```java
appsyncClient.mutationQueue.isEmpty()

appsyncClient.mutationQueue.clear()
```

Thoughts?

Reply:
I will go with the isMutationQueueEmpty method